packages/components/tests/dummy/app/templates/components/alert.hbs

```diff
@@ -0,0 +1,257 @@
+  <h3 class="dummy-h3">Component API</h3>
+  <p class="dummy-paragraph">Here is the API for the component:</p>
+
+  <dl class="dummy-component-props">
+    <dt>type <code>enum</code></dt>
+    <dd>
+      <p>Has a border and a background color. Names
+        <code class="dummy-code">global</code>
+        and
+        <code class="dummy-code">page or inline</code>
+        are temporary and subject to change.</p>
+      <p>Acceptable values:</p>
+      <ol>
+        <li class="default">global</li>
+        <li>page or inline</li>
+      </ol>
+    </dd>
+
+    <dt>isCompact <code>boolean</code></dt>
+    <dd>
+      <p>
+        <code class="dummy-code">isCompact</code>
+        is a variation on the
+        <code class="dummy-code">page or inline</code>
+        alert type ONLY. An
+        <code class="dummy-code">isCompact</code>
+        alert does NOT have a border. An
+        <code class="dummy-code">isCompact</code>
+        alert does NOT have a background color.
+      </p>
+      <p>Acceptable values:</p>
+      <ol>
+        <li>true</li>
+        <li class="default">false</li>
+      </ol>
+    </dd>
+
+    <dt>intent <code>enum</code></dt>
+    <dd>
+      <p><code class="dummy-code">intent</code>
+        result in a default
+        <code class="dummy-code">icon</code>, which can be overriden.
+        <code class="dummy-code">intent</code>
+        chooses a background color, border,
+        <code class="dummy-code">titleText</code>
+        color (if applicable), and
+        <code class="dummy-code">bodyText</code>
+        color (if applicable).
+      </p>
+      <p>Acceptable values:</p>
+      <ol>
+        <li class="default">neutral</li>
+        <li>highlight</li>
+        <li>success</li>
+        <li>warning</li>
+        <li>critical</li>
+      </ol>
+    </dd>
+
+    <dt>hasIcon <code>boolean</code></dt>
+    <dd>
+      <p>Acceptable values:</p>
+      <ol>
+        <li class="default">true</li>
+        <li>false</li>
+      </ol>
+    </dd>
+
+    <dt class="dummy-indent">icon <code>string</code></dt>
+    <dd class="dummy-indent">
+      <p>Use this parameter to show an icon. Exists if
+        <code class="dummy-code">hasIcon=true</code>.
+      </p>
+      <p>Acceptable value:</p>
+      <p>Any
+        <a href="https://flight-hashicorp.vercel.app/" target="_blank" rel="noopener noreferrer">
+          Flight icon</a>
+        name.
+      </p>
+    </dd>
+
+    <dt>titleText <code>string</code></dt>
+    <dd>
+      <p>
+        The title text in the alert.
+        <code class="dummy-code">titleText</code>
+        OR
+        <code class="dummy-code">bodyText</code>
+        is required.
+      </p>
+    </dd>
+
+    <dt>bodyText <code>string</code></dt>
+    <dd>
+      <p>The body text in the alert.
+        <code class="dummy-code">titleText</code>
+        OR
+        <code class="dummy-code">bodyText</code>
+        is required.</p>
+    </dd>
+
+    <dt>hasButton <code>boolean</code></dt>
```

Comment:
[issue] this will not work with multiple buttons/links. 

I propose this API (pseudo-TypeScript) using a `@actions` prop that is an array of objects that you can loop on:

```
action = {
    type: button | link
    icon
    iconPosition
    text
    ...attrs
    all the props supported by Button or Link(To)Standalone to execute the action (in TS we would use <pick> or <extend>)
}

actions = action[]
```

Reply:
Everything you said  ^^ makes sense, there are some changes following https://github.com/hashicorp/design-system/pull/56#discussion_r828298782 coming, but will port to the new branch

Sounds like `hasActions` doesn't necessarily have to yield a `Hds::Button` and a `Hdsl::Link::Standalone`, it's more just a div with whatever you fill in (could be 2 buttons, 2 links, other). So almost seems like `hasActions` exposes a div and what's in there is up to the user. I'll propose something shortly / see where Christian wants to land on the CRD

Reply:
let's discuss about this (`hasAction + div` vs `actions`). maybe it's not an `or` but an `and`.

Reply:
Sounds good, let's discuss "below the body" with @cveigt, because I'm getting confused about what you both want. I put in `hasActions` for now https://github.com/hashicorp/design-system/pull/92/commits/e4f4b7739c4a386e06a0bc2a7c5492fe57416251, can definitely workshop it

I proposed a time after retro if both or one of you is free, but feel free to move the time